descriptors/ModuleDescriptor-template.json

```diff
@@ -15,6 +15,16 @@
             "consortia.tenants.collection.get"
           ],
           "modulePermissions": []
+        },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/consortia/tenants",
+          "permissionsRequired": [
+            "consortia.tenants.post"
```

Comment:
We should follow the same naming conventions, in this case we should use this name: consortia.tenants.item.post

Reply:
fixed